packages/code/src/Code.tsx

```diff
@@ -140,8 +257,48 @@ function Code({
   chromeTitle = '',
   ...rest
 }: CodeProps) {
+  const scrollableMultiLine = useRef<HTMLPreElement>(null);
+  const scrollableSingleLine = useRef<HTMLDivElement>(null);
+  const [scrollState, setScrollState] = useState<ScrollState>(ScrollState.None);
+  const [copied, setCopied] = useState(false);
+
+  useEffect(() => {
+    let timeoutId: any;
+
+    if (copied) {
+      timeoutId = setTimeout(() => setCopied(false), 1500);
+    }
+
+    return () => clearTimeout(timeoutId);
+  });
```

Comment:
```suggestion
  }, [copied]);
```

We only want this to run when the value of copied changes.